### DISCUSSION TITLE
Using Herbie to make timeseries of temperature and dew point

### DISCUSSION THREAD

**Discussion by blaylockbk** [blaylockbk/Herbie#154] [Show and tell]
Notebook from Scott Collis (@scollis):
https://github.com/scollis/notebooks/blob/master/Herbie_Chicago_Locs.ipynb

<img width="446" alt="image" src="https://user-images.githubusercontent.com/6249613/212522592-d21f311a-58ec-47f8-9aa3-07c2db7fc881.png">

**Comment by scollis** [comment]
Thanks! I plan to embellish this more and submit to Pythia (cc @ktyle and @mgrover1)
